Sources/Sentry/include/HybridPublic/SentryDependencyContainer.h

```diff
@@ -101,6 +104,7 @@ SENTRY_NO_INIT
 @property (nonatomic, strong) SentryThreadInspector *threadInspector;
 @property (nonatomic, strong) SentryFileIOTracker *fileIOTracker;
 @property (nonatomic, strong) SentryCrash *crashReporter;
+@property (nonatomic, strong) SentryScopeContextPersistentStore *scopeContextStore;
```

Comment:
`m`: Please add the new properties to `testThreadSafety_ResetAndAccessProperties` to ensure testing the thread safety.